common.go

```diff
@@ -949,6 +954,9 @@ type Certificate struct {
 	// OCSPStaple contains an optional OCSP response which will be served
 	// to clients that request it.
 	OCSPStaple []byte
+	// DelegatedCredential contains a signed object containing a public
+	// key for signing handshakes
+	DelegatedCredential DelegatedCredential
```

Comment:
I understand. The Certificate struct will then contain marshalled DC and the associated PrivateKey. Both received from the delcred sub package.